.github/workflows/checks.yml

```diff
@@ -40,23 +40,45 @@ jobs:
       - name: Run SwiftFormat lint
         run: mint run swiftformat --lint .
 
-      - name: Set up Python 3.9.12
-        uses: actions/setup-python@v2
+      # TODO: Replace with installing release when release is available
+      - name: Clone starknet-devnet-rs
+        run: |
+          git clone https://github.com/0xSpaceShard/starknet-devnet-rs.git starknet-devnet-rs
+          pushd ${{ github.workspace }}/starknet-devnet-rs
+          git checkout a0d87f2f621adb6278c1bdba523ddb58268bf483
+          popd
+
+      - name: Cache devnet build
+        uses: actions/cache@v3
         with:
-          python-version: 3.9.12
+          path: starknet-devnet-rs/target/release
+          key: ${{ runner.os }}-starknet-devnet-rs-target-release-${{ hashFiles('starknet-devnet-rs/Cargo.lock') }}
```

Comment:
You mean the SHA of starknet-devnet-rs commit used on CI? `Cargo.lock` should be already unique. 
We could use either one, do you think we need both?

Reply:
yup, the code base could change without cargo lock changing. we could have both, but I guess right now - since we're pinning to specific commit - using only commit hash should be enough (as this would include whatever changes are done to .lock)